src/hera/workflows/_context.py

```diff
@@ -29,6 +29,10 @@ def enter(self, p: Subbable) -> None:
     def exit(self) -> None:
         self._pieces.pop()
 
+    @property
+    def active(self) -> bool:
+        return bool(self._pieces)
+
```

Comment:
To check if context is active. Useful in determining what the call to a script decorated function should return.